src/pages/LabEditor/LabEditor.tsx

```diff
@@ -105,6 +108,24 @@ export default function LabEditor({match: {params: {moduleId, labId}}}: Props) {
   const getFieldName = (name: keyof LabForm) => name;
 
   const [selectedVm, setSelectedVm] = useState<number|undefined>();
+  const popover = (
```

Comment:
This pull request seems to include code for the tooltip pr. Please remove this and push the code only relating to this feature so we do not have any merge conflicts.